### DISCUSSION TITLE
Using h8mail on a gzipped COMB?

### DISCUSSION THREAD

**Discussion by ForceIntoAndroi** [khast3x/h8mail#140] [Q&A] [answered]
Hello khast3x and thank you for your great work!

Im an part-time pen-testing teacher, and i have to say, there is no better tool to make my student wake up, sit straight and pay attention, as when i use your tools to reveal the students leaked passwords. And i dont even have to do it to everyone, i only have to say "so Tony, i see here that you like to choose passwords with the name of a month in them, and tack on a couple of digits....". When he reacts with chock, all of the other students realize that they should pay attention :) Extremly effective, thank you again!

Im currently using the decompressed COMB database on my laptop which works great. However i need to censor everything i show on the projector, which takes away a lot of the punch. What i would really like to do is to be able to search the COMB database on my mobile phone, so i can selectivly show the full information to one (and only one) student at a time. However, my android does not have enough space (nor able to accept a micro-sd card unfortunatly) to house the COMB database in decompressed state. 

It does however have enough space for the compressed .tar.gz file, and i have been able to run h8mail in an android terminal (i installed Pydroid 3 and installed h8mail via pip3). It seems to work just as it works on my laptop, however, when i use the -gz and the -bc flags together, there seems to be a collision. From what i gather, h8mail expect a specific hierarchy of folder and files when the flag -bc is used, but a file ending with .gz the the flag -gz is used.... 

Is there any way of using h8mail search on a gzipped version of the COMB database? If not, is there any way i could rearrange the COMB database hierarchy into a single big file, then compress it with gzip, in order for h8mail to be able to use it?

I know i dream high, but what other dreams are worth chasing?

In any case, thank you so much worth this powerful teaching aid! :)

**Comment by khast3x** [comment, marked as answer]
Hello,

Sorry for the late answer. Glad to know h8mail is useful during your class :)
The COMB uses a script called `query.sh` that searches using a decompressed version. You could compress manually the whole COMB and search with `-gz`, but it will take a long time.
You might want to rent a small VPS, and move h8mail and the COMB to it, and ssh to it with your phone, or buy a USB stick/external HDD that you can plug into your phone through MicroUSB or USB-C, and move the COMB to it.

Hope this helps!
